torch/csrc/autograd/python_engine.h

```diff
@@ -23,6 +23,7 @@ struct PythonEngine : public Engine {
       const variable_list& inputs,
       bool keep_graph,
       bool create_graph,
+      bool accumulate_grad,
```

Comment:
Could we add a comment here for what accumulate_grad does, for future code readers?

Reply:
Creating a separate PR for this